run_bot_streaming.sh

```
@@ -0,0 +1,283 @@
+            elif [[ $line == *"Virtual display started"* ]]; then
+                print_success "$line"
+            else
+                echo "$line"
+            fi
+        done
+
+    # Check if the last command was successful
+    if [ ${PIPESTATUS[0]} -eq 0 ]; then
+        print_success "Bot session completed successfully"
+        # List generated files
+        if [ -d "$output_dir" ] && [ "$(ls -A $output_dir)" ]; then
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)